src/SingleStoreConnector/Protocol/ProtocolCapabilities.cs

```diff
@@ -134,17 +134,7 @@ internal enum ProtocolCapabilities : ulong
 	QueryAttributes = 0x800_0000,
 
 	/// <summary>
-	/// Client supports progress indicator.
+	/// Client supports caching metadata for prepared statements (MARIADB_CLIENT_CACHE_METADATA).
 	/// </summary>
-	MariaDbClientProgress = 0x1_0000_0000,
-
-	/// <summary>
-	/// Client supports COM_MULTI (i.e., CommandKind.Multi)
-	/// </summary>
-	MariaDbComMulti = 0x2_0000_0000,
-
-	/// <summary>
-	/// Support of array binding.
-	/// </summary>
-	MariaDbStatementBulkOperations = 0x4_0000_0000,
+	MariaDbCacheMetadata = 0x10_0000_0000,
```

Comment:
please add a test that ensures `MariaDbCacheMetadata` and `QueryAttributes` capabilities sent by the server are set to False. A problem may arise if SingleStore uses these flags for different purposes, so we need to make sure that they are not used, so set to False.